packages/hash/frontend/theme-override.d.ts

```diff
@@ -119,5 +129,37 @@ declare module "@mui/material/Typography" {
   }
 }
 
+declare module "@mui/material/Button" {
+  interface ButtonPropsVariantOverrides {
+    transparent: true;
+    primary: true;
+    secondary: true;
+    tertiary: true;
+    tertiary_quiet: true;
+    icon: true;
```

Comment:
Why are we adding an `icon` variant to the `Button` component, instead of using the [existing `IconButton` MUI component](https://mui.com/api/icon-button/)?